Dockerfile

```diff
@@ -8,4 +8,4 @@ COPY . /opt/test-runner
 
 WORKDIR /opt/test-runner
 
-ENTRYPOINT [ "sh", "/bin/run.sh" ]
```

Comment:
@ErikSchierboom couldn't that be fixed by doing `bin/run.sh` rather than `./bin/run.sh`?

Reply:
Yep, that works. Updated!